lib/sdk/tests/index.html

```diff
@@ -0,0 +1,13 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js"></script>
+<script>
+require(
+  ["../index"],
+  function(sdk) {
+    console.log(sdk);
```

Comment:
I guess better print only versions.
```ts
const { BINARYEN_VERSION, ASSEMBLYSCRIPT_VERSION } = sdk;
console.log(
`Bynaryen v${BINARYEN_VERSION}
AssemblyScript v${ASSEMBLYSCRIPT_VERSION}`
);
```
wdyt?

Reply:
This is mostly there so one can inspect the result in a browser, including the contents of the objects. It's easy to see if something went wrong then, like if something is an empty object or doesn't have the contents one would expect.

Reply:
got it

Reply:
Made it print both, plus a few assertions :)